careers.html

```diff
@@ -0,0 +1,143 @@
+<!DOCTYPE html>
+<html lang="en">
+<head>
+  <meta http-equiv="Content-Type" content="text/html; charset=UTF-8"/>
+  <meta name="viewport" content="width=device-width, initial-scale=1, maximum-scale=1.0"/>
+  <title>Infratab Bangalore Pvt. Ltd</title>
```

Comment:
@priyanka0901 Favicon icon should be added first if you are starting any project.